desktop/assets/texts/ar-SA/filesystem_access_dialog.ftl

```diff
@@ -0,0 +1,5 @@
+filesystem-access-dialog-title = طلب الوصول إلى الملفات
+filesystem-access-dialog-message = يحاول الفيلم الإتصال بهذا الملف. هل تريد السماح به؟
```

Comment:
Does this really say "connect to this file"...? :thinking:

Reply:
I suspect that's an automatic translation error, LLMs positively detect this as a request to access the file. This is probably caused by a word having multiple meanings depending on the context, and I'm trusting the translator in this regard.